anko/library/static/commons/src/Resources.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("NOTHING_TO_INLINE", "unused")
+package org.jetbrains.anko
+
+import android.content.Context
+
+fun Context.int(@IntegerRes resource: Int): Int = resources.getInteger(resource)
```

Comment:
Perhaps a different name that is a little bit more descriptive of what it's getting for you? Something like `resource()` or something?

Reply:
What do you mean? It is getting an `IntegerRes`. The name of the function for getting a `DimensionRes` is `dimen()`, so what is wrong with `int()`?

Reply:
Oh, sorry, I totally misread the function. Carry on, nothing to see here. :roll_eyes: